src/test/java/com/azure/cosmos/kafka/connect/sink/integration/SinkConnectorIT.java

```diff
@@ -48,15 +57,23 @@
 @Category(IntegrationTest.class)
 public class SinkConnectorIT {
     private static Logger logger = LoggerFactory.getLogger(SinkConnectorIT.class);
-    
+
     private String databaseName;
-    private String topic;
     private String connectorName;
     private Builder connectConfig;
     private CosmosClient cosmosClient;
     private CosmosContainer targetContainer;
     private KafkaConnectClient connectClient;
     private KafkaProducer<String, JsonNode> producer;
+    private KafkaProducer<GenericRecord, GenericRecord> avroProducer;
+    private String kafkaTopicJson;
+    private static final String KAFKA_TOPIC_JSON_SCHEMA = "sink-test-json-schema";
```

Comment:
Nit but would be good to group the statics together and separate from these variables